docs/product-installation-guide/src/main/asciidoc/clustering-ps-configure-domain-proc.adoc

```diff
@@ -22,7 +22,7 @@ By using domain mode with a cluster, you can centrally manage and publish config
 
 . Edit the `data-stores` property and the `timer-service thread-pool-name` above it.
 +
-The `datasource-jndi-name` is the JNDI name of the database specified at the beginning of this procedure. You can enter any name for the value of the `partition` property. Replace the  `default-data-store` attribute value with `ejb_timer_ds`· Set the value of `refresh-interval` to specify how often the EJB timer connects to the database to synchronize and load tasks to be processed:
+The `datasource-jndi-name` is the JNDI name of the database specified at the beginning of this procedure. You can enter any name for the value of the `partition` property. Replace the  `default-data-store` attribute value with `ejb_timer_ds`· Set the value of `refresh-interval` to specify how often in milliseconds the EJB timer connects to the database to synchronize and load tasks to be processed:
 +
```

Comment:
I think this can be improved: "You can enter any name for the value of the partition property." 

https://docs.jboss.org/author/display/WFLY9/EJB3+Clustered+Database+Timers states:

> A node will only see timers from other nodes that have the same partition name. This allows you to break a large cluster up into several smaller clusters, which should improve performance. e.g. instead of having a cluster of 100 nodes, where all hundred are trying to execute and refresh the same timers, you can create 20 clusters of 5 nodes by giving ever group of 5 a different partition name.